_posts/atoms/2018-09-07-icons.html

```diff
@@ -8,14 +8,222 @@
              and [more](https://fontawesome.com/how-to-use/on-the-web/styling/).
 ---
 
-`Add`:
-%i.fas.fa-plus-circle.text-primary
-Add
+%h3 Icon Modifiers
+
+Rotate with `fa-spin`:
+%i.fas.fa-sync-alt.fa-spin
+%br
+
+Rotate on click with `fa-spin`:
+%i.fas.fa-sync-alt.rotate-on-click
+:javascript
+  // Example to show how this can be achieved
+  $('i.rotate-on-click').click(function() {
+    element = $(this);
+    element.addClass('fa-spin');
+    // Sleep for 1.5 second... this would be your AJAX call or whatever needs to be done in the refresh
+    setTimeout(function() {
+      element.removeClass('fa-spin');
+    }, 1500);
+  });
+
+%h5 Colors
+
+%p
+  Some icons have a predefined color to better convey information. For example, icons used in the context
+  of a destructive action, which could have severe consequences, are displayed in red to express danger.
+
+%p
+  For icons which don't have a predefined color on this page, use
+  %a{ href: 'colors.html' } colors
+  to convey information based on the context.
+
+%h5 Sizes
+
+%p
+  We only use a subset of the
+  %a{ href: 'https://fontawesome.com/how-to-use/on-the-web/styling/sizing-icons' }icon sizes provided by FontAwesome
+
+%i.fas.fa-headphones.fa-xs
+%i.fas.fa-headphones.fa-sm
+%i.fas.fa-headphones
+%i.fas.fa-headphones.fa-lg
+%i.fas.fa-headphones.fa-2x
+%i.fas.fa-headphones.fa-3x
+
+%h3 Places
+
+%p These icons are for the `Places` links displayed in the left navigation sidebar.
+
+`All Projects`:
+%i.fas.fa-list
+%br
+
+`Configuration`:
+%i.fas.fa-cogs
+%br
+
+`Status Monitor`:
+%i.fas.fa-heartbeat.text-danger
+%br
+
+`Tasks`:
+%i.fas.fa-tasks
+%br
+
+`Your Home Project`:
+%i.fas.fa-cubes
+
+%h3 Actions
+
+%p
+  These icons are for generic actions like `Create` or `Delete`.
+  They aren't limited to the `Actions` links in the left navigation sidebar, they are used in various contexts of the application.
+
+`Branch`:
+%i.fas.fa-code-branch
+Branch
+\/
+%i.fas.fa-code-branch{ title: 'Branch' }
+*(inline)*
```

Comment:
What do you want to express with the *inline* thing?

Reply:
It's to show how to use an icon _inline_. I guess I went a bit over the top by doing it for every icon.

Reply:
This is now gone.